src/run.rs

```diff
@@ -0,0 +1,48 @@
+use chrono::prelude::*;
+use clap::ArgMatches;
+
+use crate::util::file::*;
+
+fn get_day(matches: &ArgMatches) -> u32
+{
+    if let Some(day) = matches.get_one::<String>("day")
+    {
+        day.parse().unwrap()
+    }
+    else
+    {
+        Utc::now().day()
+    }
+}
+
+pub async fn run(matches: &ArgMatches) -> Result<(), Box<dyn std::error::Error>>
+{
+    let day = get_day(matches);
+
+    let cwd = std::env::current_dir()?;
+    let path = cargo_path(&cwd).await.unwrap_or(cwd);
+    let dir = day_path(path, day).await?;
+    let dir = dir.to_str().unwrap();
+
+    // check here that 'input' actually exists
+    // ...
+
+    let res = tokio::process::Command::new("cargo")
+        .args(["run", "--bin", format!("day_{:02}", day).as_str()])
```

Comment:
We need
```rs
.args(["run", "--bin", format!("day_{:02}", day).as_str(), "--color", "always"])
```
here so we can keep the color from stdout when printing later. Otherwise it will not print with colors and it will be harder to read:(

Reply:
Good catch